2024-08-09 11:25:59,569 ERROR: Ocorreu um erro ao fazer o upload: I/O operation on closed file. [in C:\Users\T_J\PycharmProjects\ideal-octo-engine\app\routes.py:75]
2024-08-09 11:26:13,571 ERROR: Ocorreu um erro ao fazer o upload: I/O operation on closed file. [in C:\Users\T_J\PycharmProjects\ideal-octo-engine\app\routes.py:75]
2024-08-09 11:27:59,246 ERROR: Ocorreu um erro ao fazer o upload: I/O operation on closed file. [in C:\Users\T_J\PycharmProjects\ideal-octo-engine\app\routes.py:75]
2024-08-09 15:21:24,205 ERROR: HTTP error occurred: 401 Client Error: Unauthorized for url: https://api.openpix.com.br/api/v1/charge [in C:\Users\T_J\PycharmProjects\ideal-octo-engine\app\routes.py:163]
2024-08-09 15:28:28,587 ERROR: HTTP error occurred: 401 Client Error: Unauthorized for url: https://api.openpix.com.br/api/v1/charge [in C:\Users\T_J\PycharmProjects\ideal-octo-engine\app\routes.py:163]
2024-08-09 15:28:42,243 ERROR: HTTP error occurred: 401 Client Error: Unauthorized for url: https://api.openpix.com.br/api/v1/charge [in C:\Users\T_J\PycharmProjects\ideal-octo-engine\app\routes.py:163]
2024-08-09 15:35:30,821 ERROR: Other error occurred: 'paymentRequest' [in C:\Users\T_J\PycharmProjects\ideal-octo-engine\app\routes.py:166]
2024-08-09 15:37:40,480 ERROR: Erro de valor: Campo 'paymentRequest' não encontrado na resposta [in C:\Users\T_J\PycharmProjects\ideal-octo-engine\app\routes.py:170]
2024-08-09 15:37:54,677 ERROR: Erro de valor: Campo 'paymentRequest' não encontrado na resposta [in C:\Users\T_J\PycharmProjects\ideal-octo-engine\app\routes.py:170]
2024-08-09 15:40:04,545 ERROR: Erro de valor: Campo 'paymentRequest' não encontrado na resposta [in C:\Users\T_J\PycharmProjects\ideal-octo-engine\app\routes.py:170]
2024-08-09 15:40:12,712 ERROR: Erro de valor: Campo 'paymentRequest' não encontrado na resposta [in C:\Users\T_J\PycharmProjects\ideal-octo-engine\app\routes.py:170]
2024-08-09 16:16:01,714 ERROR: HTTP error occurred: 404 Client Error: Not Found for url: https://api.openpix.com.br/api/v1/qrcode/pix/66b3ad1b5669dcd626dd9685 [in C:\Users\T_J\PycharmProjects\ideal-octo-engine\app\routes.py:167]
2024-08-09 16:16:13,007 ERROR: HTTP error occurred: 404 Client Error: Not Found for url: https://api.openpix.com.br/api/v1/qrcode/pix/66b3ad1b5669dcd626dd9685 [in C:\Users\T_J\PycharmProjects\ideal-octo-engine\app\routes.py:167]
2024-08-09 16:22:25,383 ERROR: HTTP error occurred: 404 Client Error: Not Found for url: https://api.openpix.com.br/api/v1/qrcode/pix/66b3ad1b5669dcd626dd9685 [in C:\Users\T_J\PycharmProjects\ideal-octo-engine\app\routes.py:164]
2024-08-09 16:29:33,746 ERROR: HTTP error occurred: 404 Client Error: Not Found for url: https://api.openpix.com.br/api/v1/qrcode/pix/66b3ad1b5669dcd626dd9685 [in C:\Users\T_J\PycharmProjects\ideal-octo-engine\app\routes.py:164]
2024-08-09 16:29:43,574 ERROR: HTTP error occurred: 404 Client Error: Not Found for url: https://api.openpix.com.br/api/v1/qrcode/pix/66b3ad1b5669dcd626dd9685 [in C:\Users\T_J\PycharmProjects\ideal-octo-engine\app\routes.py:164]
2024-08-09 16:30:52,072 ERROR: HTTP error occurred: 404 Client Error: Not Found for url: https://api.openpix.com.br/api/v1/qrcode/pix/ [in C:\Users\T_J\PycharmProjects\ideal-octo-engine\app\routes.py:164]
2024-08-09 16:31:11,367 ERROR: HTTP error occurred: 404 Client Error: Not Found for url: https://api.openpix.com.br/api/v1/qrcode/pix/ [in C:\Users\T_J\PycharmProjects\ideal-octo-engine\app\routes.py:164]
2024-08-09 16:37:09,423 ERROR: HTTP error occurred: 404 Client Error: Not Found for url: https://api.openpix.com.br/api/v1/qrcode/pix/66b3ad1b5669dcd626dd9685 [in C:\Users\T_J\PycharmProjects\ideal-octo-engine\app\routes.py:169]
2024-08-09 16:37:30,711 ERROR: HTTP error occurred: 404 Client Error: Not Found for url: https://api.openpix.com.br/api/v1/qrcode/pix/66b3ad1b5669dcd626dd9685 [in C:\Users\T_J\PycharmProjects\ideal-octo-engine\app\routes.py:169]
2024-08-09 17:07:06,964 ERROR: Other error occurred: 'status' is an invalid keyword argument for Transaction [in C:\Users\T_J\PycharmProjects\ideal-octo-engine\app\routes.py:172]
2024-08-09 17:09:19,910 ERROR: Other error occurred: 'status' is an invalid keyword argument for Transaction [in C:\Users\T_J\PycharmProjects\ideal-octo-engine\app\routes.py:172]
